ads/doubleclick.js

```diff
@@ -60,9 +60,14 @@ export function doubleclick(global, data) {
       }
 
       pubads.addEventListener('slotRenderEnded', function(event) {
+        let creativeId = event.creativeId ||
```

Comment:
oh yeah, but wouldn't you have coerced the value at that point to a string because of the concat? i guess i was wondering if `event.creativeId` for some reason was a `0`. feel free to disregard this comment if that can never happen